lib/options.js

```diff
@@ -90,6 +90,13 @@ module.exports = {
         // autopadding character - empty in case autopadding is disabled
         options.autopaddingChar = options.autopadding ? mergeOption(opt.autopaddingChar, '   ') : '';
 
+        // auto expand bar width to visible window width
+        options.autoExpand = mergeOption(opt.autoExpand, false);
+
+        // necessary for auto expand and dyanmic window resize
+        options.barCompleteChar = opt.barCompleteChar || '=';
```

Comment:
not required in case the pre-computed string sequences are used

Reply:
See above, on static view you are right, but we don't knwo the maximum window size, the window could be resized.